docs/faq.md

```diff
@@ -15,6 +15,11 @@ Here's an exhaustive list of all the major changes introduced in Helm 3.
 In Helm 3, Helm switched the Go import path over from `k8s.io/helm` to `helm.sh/helm`. If you intend
 to upgrade to the Helm 3 Go client libraries, make sure to change your import paths.
 
+### Capabilities
+
+Capabilities built-in has been simplified.
```

Comment:
Would you mind documenting what fields changed, and to what? e.g. `.Capabilities.KubeVersion` to `.Capabilities.Version`. This is a major breaking change to existing charts so we should document the changes required to existing charts.

Reply:
updated to be backwards compatible

Reply:
Some of the field names did change (for the better) so we should probably still list out what fields changed, and where. For example:

- `.Capabilities.KubeVersion.GitVersion` -> `.Capabilities.KubeVersion.Version`

I don't believe the platform/runtime fields were flattened either. e.g.

- `.Capabilities.KubeVersion.Platform` -> `.Capabilities.Platform`

Are you planning to add those back?

Reply:
I agree with @bacongobbler that it would be good to show the changes. Easy for user to see then.

Reply:
Platform was a garbage field that was sentenced to collection.

Some fields we never populated with relevant information.  I deleted all the no-op fields.

@bacongobbler `GitVersion` still returns the same value.  `.Capabilities.KubeVersion.GitVersion == .Capabilities.KubeVersion.Version`